### PR TITLE
Correct the element name

### PR DESCRIPTION
**Pull request by Snoopy1866**
### Description

Accroding to the html source code example, the outer element is `<article>` not `<div>`.

### CODE REVIEW

File: files/en-us/web/css/@scope/index.md [hunk 1]
```diff
@@ -423,8 +423,8 @@ img {
...
-- The second `@scope` block also defines its scope root as elements with a class of `.feature`, but in addition defines a scope limit of `figure`. This ensures that contained rulesets will only be applied to matching elements within the scope root (`<div class="figure"> ... </div>` in this case) that **are not** nested inside descendant `<figure>` elements. This `@scope` block contains a single ruleset that styles `<img>` elements with a thick black border and a golden background color.
+- The first `@scope` block defines its scope root as elements with a class of `.feature` (in this case, the outer `<article>` only), demonstrating how `@scope` can be used to theme a specific HTML subset.
+- The second `@scope` block also defines its scope root as elements with a class of `.feature`, but in addition defines a scope limit of `figure`. This ensures that contained rulesets will only be applied to matching elements within the scope root (`<article class="figure"> ... </article>` in this case) that **are not** nested inside descendant `<figure>` elements. This `@scope` block contains a single ruleset that styles `<img>` elements with a thick black border and a golden background color.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@scope/index.md:427]:
```suggestion
- The second `@scope` block also defines its scope root as elements with a class of `.feature`, but also defines a scope limit of `figure`. This ensures that contained rulesets will only be applied to matching elements within the scope root (`<article class="feature"> ... </article>` in this case) that **are not** nested inside descendant `<figure>` elements. This `@scope` block contains a single ruleset that styles `<img>` elements with a thick black border and a golden background color.
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks for your contribution to MDN, @Snoopy1866! Your changes are correct, but I've just made one small suggestion that fixes an incorrect `class` name and makes a small language suggestion.

Accept it if you agree.

Thanks again!

**Comment by Snoopy1866** [comment]
Agree!

**Review by chrisdavidmills — APPROVED** [review]
Perfect, let's get this merged. Thanks again for your contribution @Snoopy1866, and wishing you all the best.

**Pull request merged**